go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.3 // indirect
 	golang.org/x/crypto v0.0.0-20190103213133-ff983b9c42bc // indirect
+	golang.org/x/net v0.0.0-20190108225652-1e06a53dbb7e
```

Comment:
`golang.org/x/net` already removed in later PR after you created the branch, you do not need to add it back.